dashboard/app/controllers/registrations_controller.rb

```diff
@@ -51,6 +51,26 @@ def create
     end
   end
 
+  def destroy
+    # TODO: (madelynkasula) Remove this flag when the new account
+    # deletion UI is available for all users.
```

Comment:
It might be helpful for this comment to mention the `ACCOUNT_DELETION_NEW_FLOW` experiment flag by name, so when we are tearing out that flag we also find this comment.

Reply:
good call. adding that now